rotkehlchen/api/rest.py

```diff
@@ -4446,11 +4446,9 @@ def get_evm_counterparties_details(self) -> Response:
         removing duplicates.
         """
         return api_response(
-            result=process_result(
-                _wrap_in_ok_result(
-                    result=list(self.rotkehlchen.chains_aggregator.get_all_counterparties()),
-                ),
-            ),
+            result=process_result(_wrap_in_ok_result(
```

Comment:
what did you do here? What changed?

Reply:
before the image should exist.
now it can return either image or icon